pkg/lib/bundle/generate.go

```diff
@@ -146,7 +146,7 @@ func GenerateFunc(directory, outputDir, packageName, channels, channelDefault st
 	} else if err != nil {
 		return err
 	} else {
-		log.Info("A bundle.Dockerfile already exists in current working directory")
+		log.Printf("A bundle.Dockerfile already exists in current working directory: %s", workingDir)
```

Comment:
```suggestion
		log.Infof("A bundle.Dockerfile already exists in current working directory: %s", workingDir)
```
You can keep the log level with `Infof`

Reply:
Done.